app/src/main/res/layout/item_start_ride_bottom_sheet.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'd rename this and `item_scooter_bottom_sheet` to something like `dialog_start_ride` / `dialog_scooter_details`